### ISSUE TITLE
In karibu testing 2.1.5 MockRequest was removed. Why and what is the replacement?

### ISSUE THREAD

**Issue by satorstefan** [mvysny/karibu-testing#170]
import com.github.mvysny.kaributesting.mockhttp.MockRequest;

Here the class is still there:
![image](https://github.com/mvysny/karibu-testing/assets/25597168/d0056f0e-8123-4eb2-8a1c-22777d1647cd)

Thats how I use it currently:
![image](https://github.com/mvysny/karibu-testing/assets/25597168/df6580f9-cfdf-4c94-a3b3-c949df69817c)

**Comment by satorstefan** [comment]
![image](https://github.com/mvysny/karibu-testing/assets/25597168/7496b0c5-1da3-4430-8e65-7294b81436d5)

**Comment by TheHaf-at-FraunhoferIOSB** [comment]
For some reason, that dependency mock-servlet-environment5 is not (yet) available in version 2.1.5. You could try adding the dependency in version 2.1.4 to your project explicitely.
Personally I am just staying on version 2.1.4 for the time being.

**Comment by mvysny** [comment]
That's true. The mock-servlet-environment5 and mock-servlet-environment held fake classes implementing bits of jakarta.servlet and javax.servlet respectively. They were almost identical yet present in two distinct KaribuTesting branches, master and 1.4.0, duplicating maintenance efforts.

I've extracted the library to https://github.com/mvysny/fake-servlet which KaribuTesting pulls in as a dependency. The classes were renamed to FakeSession and such, however there should still be deprecated typealiases for MockSession classes.

In short, the mock-servlet-environment5 dependency was replaced by the fake-servlet5 dependency.
